process/metadata.csv

```diff
@@ -21,3 +21,4 @@ system.processes.voluntary_ctx_switches,gauge,,event,,The number of voluntary co
 system.processes.run_time.avg,gauge,,second,,The average running time of all instances of this process,0,system,avg run time
 system.processes.run_time.max,gauge,,second,,The longest running time of all instances of this process,0,system,max run time
 system.processes.run_time.min,gauge,,second,,The shortest running time of all instances of this process,0,system,min run time
+system.processes.cpu.normalized_pct,gauge,,percent,,The normalized process CPU utilization.,0,system,processes cpu normalized pct
```

Comment:
Super small nitpick, but can we move this to the top under `system.processes.cpu.pct`, and can we change the description to something like:
>The normalized CPU utilization of a process

And we can also change the description of `system.processes.cpu.pct` to:
>The CPU utilization of a process